torch/quantization/qconfig.py

```diff
@@ -64,7 +64,7 @@ def __new__(cls, activation=torch.nn.Identity, weight=torch.nn.Identity):
 
 default_dynamic_qconfig = QConfigDynamic(activation=default_dynamic_quant_observer,
                                          weight=default_weight_observer)
-float16_dynamic_qconfig = QConfigDynamic(activation=PlaceholderObserver.with_args(dtype=torch.float16),
+float16_dynamic_qconfig = QConfigDynamic(activation=PlaceholderObserver.with_args(dtype=torch.float32),
```

Comment:
just curious, what's the context for this change?

Reply:
this is incorrect previously I think, float16_dynamic quant is taking fp32 activation as input before